django/db/backends/mysql/creation.py

```diff
@@ -11,3 +16,39 @@ def sql_table_creation_suffix(self):
         if test_settings['COLLATION']:
             suffix.append('COLLATE %s' % test_settings['COLLATION'])
         return ' '.join(suffix)
+
+    def _clone_test_db(self, number, verbosity, keepdb=False):
+        qn = self.connection.ops.quote_name
```

Comment:
What happens if more than one connection points to the same DB instance?  In our case, we have two connections "DEFAULT" and "READ_ONLY", but in our test environment, "READ_ONLY" points at the same mysql db instance.  

Is there some way to prevent reloading the fixture if dbname/host/port has already been cloned?